ngraph/test/files/paddlepaddle/models/models.csv

```diff
@@ -0,0 +1,22 @@
+assign_value_fp32,
+bilinear_downsample_false_0,
+bilinear_downsample_false_1,
+bilinear_downsample_true_0,
+bilinear_upsample_false_0,
+bilinear_upsample_false_1,
+bilinear_upsample_true_0,
+fill_constant,
+fill_constant_batch_size_like,
+nearest_downsample_false_0,
+nearest_upsample_false_0,
+pad3d_test1,
+pad3d_test2,
+slice,
+slice_1d,
+split_test1,
+split_test2,
+squeeze_null_axes,
+unsqueeze,
+yolo_box_test1,
+flatten_contiguous_range_test1,
```

Comment:
it was alphabetical order of models until last 2 items

Reply:
will change in alphabet order